journal-writings/modernized/16-various-subjects.adoc

```diff
@@ -0,0 +1,505 @@
+take such methods in education, as may prove a great injury to the minds of our children.
+
+It is a lovely sight to behold innocent children-;
+and when they are sent to schools where their tender minds are
+in imminent danger of being led astray by tutors,
+who do not live a self-denying life,
+or by the conversation of children who do not live in innocence,
+it is a case much to be lamented.
+
+While a pious tutor has the charge of no more children than he can take due care of,
+and keeps his authority in the Truth, the good spirit in which he leads and governs,
+works on the minds of such who are not hardened,
+and his labors not only tend to bring them forward in outward learning,
+but to open their understanding with respect to the true Christian life.
+But where a person has charge of too many,
+and his thoughts and time are so much employed in the outward affairs of his school,
+that he does not so weightily attend to the spirit and conduct of each individual,
+as to be enabled to administer rightly to all in due season;
+through such omissions he not only suffers, as to the state of his own mind,
+but the minds of the children are in danger of suffering also.
+
+To watch the spirit of children, to nurture them in Gospel love,
+and labor to help them against that which would mar the beauty of their minds,
+is a debt we owe them; and a faithful performance of our duty,
+not only tends to their lasting benefit, and our own peace,
+but also to render their company agreeable to us.
+
+Instruction thus administered,
+reaches the pure witness in the minds of children who are not hardened,
+and begets love in them toward those who thus lead them on;
+but where too great a number are committed to a tutor, and he, through much cumber,
+omits a careful attention to the minds of the children,
+there is a danger of disorders gradually increasing amongst them,
+until the effects thereof appear in their conduct, too strong to be easily remedied.
+
+A care has lived on my mind, that more time might be employed by parents at home,
+and by tutors at school,
+in weightily attending to the spirit and inclinations of children,
+and that we may so lead, instruct and govern them, in this tender part of life,
+that nothing may be omitted which is in our power,
+to help them on their way to become the children of our Father, who is in heaven.
+
+Meditating on the situation of schools in our provinces,
+my mind has at times been affected with sorrow,
+and under these exercises it has appeared to me,
+that if those who have large estates were faithful stewards, and laid no rent,
+or interest, or other demand, higher than is consistent with universal love;
+and those in lower circumstances would, under a moderate employ,
+shun unnecessary expense, even to the smallest article,
+and all unite in humbly seeking to the Lord, he would graciously instruct us,
+and strengthen us,
+to relieve the youth from various snares in which many of them are entangled.
+
+
```

Comment:
You don't need to change this, because it doesn't matter, but my style generally is to avoid multiple consecutive empty lines